packages/@aws-cdk/app-delivery/package.json

```diff
@@ -63,6 +65,8 @@
   ],
   "peerDependencies": {
     "@aws-cdk/aws-codepipeline-api": "^0.16.0",
+    "@aws-cdk/aws-cloudformation": "^0.16.0",
```

Comment:
You'll want to make sure you merge in the `0.17.0` version bump & update those :)